manifests/t/TUG/TeXLive/2019/TUG.TeXLive.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TUG.TeXLive
+PackageVersion: 2019
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe

```

```suggestion
  InstallerUrl: https://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe
```
Depending on which country someone downloads this application from, you'll receive a different hash which is why it uses `ftp.ntua.gr`.